extensionsGallery.json

```diff
@@ -2961,6 +2961,66 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "61",
+					"extensionName": "Visualization",
+					"displayName": "Visualization",
+					"shortDescription": "Visualization of databases",
+					"publisher": {
+						"displayName": "Jens Hunt",
+						"publisherId": "R0tenur",
+						"publisherName": "R0tenur"
+					},
+					"versions": [
+						{
+							"version": "0.0.2",
+							"lastUpdated": "2/2/2020",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
```

Comment:
Remove the VSIX package link - for external extensions we just provide a link to the download page